.gitignore

```diff
@@ -6,6 +6,7 @@ jsl.conf
 stash*.txt
 stash*.diff
 *.jsX
+.vscode/
```

Comment:
@siddharthvp: [Found it!](https://github.com/azatoth/twinkle/pull/642#discussion_r287580727)